Applications/Terminal/Terminal.h

```diff
@@ -164,6 +173,7 @@ class Terminal final : public GFrame {
     };
     EscapeState m_escape_state { Normal };
     Vector<byte> m_parameters;
+    bool m_question_param;
```

Comment:
As mentioned above I think, I'd prefer to see this removed as a member (and instead, passed as a parameter).